cartridges/int_cloudinary/cartridge/scripts/service/cldMetadata.js

```diff
@@ -64,9 +64,52 @@ function fetchMetadata() {
     return metadataFields;
 }
 
+
+/**
+ * This method uses the service to create metadata fields in cld.
+ *
+ * @param {Array} schema - Schema to be created in Cloudinary
+ * @returns {string} result - The API service response (JSON)
+ */
+function createMetadataSchemeCld(schema) {
+    var cldResponse = { ok: true, message: '', result: {} };
+    var configArgs = {};
+    var result = null;
+
+    configArgs.method = 'POST';
+    configArgs.endPoint = '/metadata_fields';
+
+    try {
+        var service = cldWebService.getService(Prefs.CLD_METADATA_UPLOAD, cldWebService.getServiceConfigs(configArgs));
+        schema.forEach(function (field) {
+            result = service.call(field);
+            if (result.ok && result.error === 0) {
+                cldResponse.ok = true;
+                cldResponse.result = result.object;
+                logger.info('Metadata fields: {0} is created successfully.', result.object.label);
+            } else {
+                if (result.error === Prefs.ERROR_CODES.UNAUTHORIZED) {
+                    logger.error('Error occurred while connecting to the service due to invalid credentials, message: {0}', result.errorMessage);
+                } else {
+                    logger.error('Error occurred while creating metadata fields from cloudinary: {0}', JSON.parse(result.errorMessage).error.message);
+                }
+                cldResponse.ok = false;
+                cldResponse.message = result.errorMessage;
```

Comment:
This will override all errors, but the last. Please concatenate as in the example in the JIRA